### DISCUSSION TITLE
Name Space for KEDA HTTP add-on and HTTPScaledObject

### DISCUSSION THREAD

**Discussion by Ravikumar-BM** [kedacore/http-add-on#506] [Q&A] [answered]
Do we need to have KEDA HTTP add-on and HTTPScaledObject should be on same namespace ?  

can I have, say example KEDA HTTP add-on is in $Namespace1 and HTTPScaledObject is in $Namespace2  ?

**Comment by tomkerkhove** [comment, marked as answer]
No, you can create them in seperate namespaces

**Reply by Ravikumar-BM** [reply]
Thanks for the confirmation

**Reply by tomkerkhove** [reply]
Happy to help!
